CMakeLists.txt

```
@@ -195,3 +195,11 @@ ADD_SUBDIRECTORY(src)
 include("${CMAKE_CURRENT_SOURCE_DIR}/cmake/setup.py.cmake")
 configure_file("cmake/config.py.in" "${CMAKE_CURRENT_BINARY_DIR}/cmake/config.py")
 INSTALL(FILES "${CMAKE_CURRENT_BINARY_DIR}/cmake/config.py" DESTINATION "${PYTHON_DEST}/sirf")
+set(SHARE_DIR ${CMAKE_INSTALL_PREFIX}/share/SIRF-${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH})
```

```suggestion
set(SHARE_DIR ${CMAKE_INSTALL_PREFIX}/share/SIRF-${VERSION_MAJOR}.${VERSION_MINOR})
```
no patch info (data and doc shouldn't really change (a lot) between patches